lib/pharos/phases/configure_master.rb

```diff
@@ -4,10 +4,10 @@ module Pharos
   module Phases
     class ConfigureMaster < Pharos::Phase
       title "Configure master"
-
+      KUBE_DIR = '/etc/kubernetes'
       PHAROS_DIR = '/etc/pharos'
+      SHARED_CERT_FILES = %w(ca.crt ca.key sa.key sa.pub).freeze
       AUTHENTICATION_TOKEN_WEBHOOK_CONFIG_DIR = '/etc/kubernetes/authentication'
-
       AUDIT_CFG_DIR = (PHAROS_DIR + '/audit').freeze
 
       def client
```

Comment:
The `client` and `kubeadm_configmap` are no longer used, maybe remove them?

Although I think we may still want to look at the kubeadm configmap in the future, in order to figure out if things like `etcd` have changed... however, the implementation needs fixing because of #140: this assumes that the `ConfigureClient` phase was run on the same `pharos-cluster` machine earlier, so upgrades would break if you ran `pharos-cluster` from a fresh machine that had never been run before.

Reply:
This should also fix #140 now